test/library/packages/Sort/RadixSort/flagsortMSB.chpl

```diff
@@ -0,0 +1,142 @@
+
+module flagsortMSB {
+  use Sort;
+  use Time;
+  use BitOps;
+  use Random;
+  config param bucketBits=8;
+  param numBuckets = (1 << bucketBits);
+  param DISTRIBUTE_BUFFER = 1;
+  config const tryUnroll = false;
+
+   // To find a position where an element's count is to be updated in the counts array.
+   private proc findPosition(place:int, element:int, numBuckets:int) {
+     var position = ((element>>place) & (numBuckets-1));
+     return position;
+   }
+
+   // To find maximum element in an array
+   private proc findMaxElement(array:[] int) {
+     return max reduce array;
+   }
+
+   // To find the place value from which msb sorting starts
+   private proc findPlace(array:[] int) {
+
+     const maxEl = findMaxElement(array);    
+     var lz = clz(maxEl);
+     var numBits = 64 - lz ;
+     var place = ((numBits)>>3)<<3;
+
+     return place;
+   }
+  
+   // MSB initial call - helper function
```

Comment:
Note that it's good habit to use [`chpldoc`-style](https://chapel-lang.org/docs/master/primers/chpldoc.doc.html) comments, especially for features that are part of the interface.